src/services/schemas/SchemasService.ts

```diff
@@ -67,6 +67,7 @@ export class SchemaService {
         error => deferred.reject(error))
       .then(error => {
           if (isNotNullOrUndefined(error)) {
+            this.unregisterProtoSchema(container, schema.fileName);
```

Comment:
So at present this is an async action that we fire and forget, no action is taken if the unregister fails/timesout. Do we want to retry if it fails? Or at least notify the user that the schema was created, but with errors and it hasn't been possible to undo its creation